lib/rtcpeerconnection/safari.js

```diff
@@ -128,6 +129,14 @@ function SafariRTCPeerConnection(configuration) {
     }
   });
 
+  // NOTE(syerrapragada): When the OrderedTrackMatcher in twilio-video.js accesses the underlying
```

Comment:
@syerrapragada Since OrderedTrackMatcher is out of this repository's scope, can you change this to:
```
This ensures that SafariRTCPeerConnection's "remoteDescription", when accessed
in an RTCTrackEvent listener, will point to the underlying RTCPeerConnection's
"remoteDescription". Before this fix, this was still pointing to "_pendingRemoteOffer"
even though a new remote RTCSessionDescription had already been applied.
```